### DISCUSSION TITLE
Wyze CAM OG & TelePhoto hardware

### DISCUSSION THREAD

**Discussion by rickgitdone** [gtxaspec/wz_mini_hacks#471] [Q&A]
Has anyone tried to get these new CAMs setup using the wyze_hacks??  they are $20 per cam ...

**Comment by rickgitdone** [comment]
I see this was already answered ...

**Comment by mach7** [comment]
@gtxaspec can we remove discussions like this where the original author asks and answers in this manner? I'll bet this has been opened to be viewed by a lot of people just to navigate away with a little disappointment that it didn't contain any relevance to any update on the topic or reference to the other answer elsewhere.
